platform/commonUI/general/src/controllers/ContextMenuController.js

```diff
@@ -36,12 +36,19 @@ define(
          * @memberof platform/commonUI/general
          * @constructor
          */
-        function ContextMenuController($scope) {
+        function ContextMenuController($scope, policyService) {
+
+            function applyPolicy(action) {
+                return policyService.allow('action', action,
+                    { domainObject: $scope.domainObject });
+            }
+
             // Refresh variable "menuActions" in the scope
             function updateActions() {
-                $scope.menuActions = $scope.action ?
-                        $scope.action.getActions({ category: 'contextual' }) :
-                        [];
+                var actions = $scope.action ?
+                    $scope.action.getActions({ category: 'contextual' }) : [];
+
+                $scope.menuActions = actions.filter(applyPolicy);
```

Comment:
@VWoeltjen Apparently not :smiley: Didn't even realize there was a decorator being applied here!